src/free_addresses/free_addresses.go

```diff
@@ -0,0 +1,181 @@
+package free_addresses
+
+import (
+	"encoding/json"
+	"encoding/xml"
+	"fmt"
+	"net"
+
+	"github.com/go-openapi/strfmt"
+	"github.com/ori-amizur/introspector/src/util"
+	"github.com/pkg/errors"
+
+	"github.com/filanov/bm-inventory/models"
```

Comment:
I don't think that it is correct to import bm-inventory models dir
Maybe we should use the swagger to generate client?
@filanov what do you think?

Reply:
We use the model.  The client wouldn't help us.
Since it is stored as a string, the encode/decode the json is to use the same model.

Reply:
The client generation is done in BM-inventory as well. 
I think that client generation generate the models as well but we don't support multi versions anyway so not sure if there is a point to do it. 
How do you going to sync the versions? how to checkout the file from there?

Reply:
The client generation is done only in bm-inventory.
In the agent we do "go get"